packages/zoltan2/test/connectivity/Zoltan2_IceUtil.h

```diff
@@ -0,0 +1,187 @@
+#ifndef __Zoltan2_ice_util_h__
+#define __Zoltan2_ice_util_h__
+
+#include<iostream>
+#include<fstream>
+#include<sstream>
+#include<string>
+
+#include "graph.h"
```

Comment:
I thought the name of this include file changed.  Would an application need to include it?

Reply:
This include was overlooked after I changed the name of the file. It will be removed.